src/Akeneo/Pim/Automation/DataQualityInsights/front/src/application/component/ProductEditForm/TabContent/DataQualityInsights/AttributesGroupsHelper.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import styled from 'styled-components';
+import {AttributeGroup, AttributeGroupCollection} from "@akeneo-pim-community/settings-ui/src/models";
+
+const translate = require('oro/translator');
+
+const Helper = styled.div`
+  margin: 5px 0 10px 0;
+  background-color: #f5f9fc;
+  padding: 10px 0px 10px 0;
+  display: flex;
+  
+  a {
+    color: #9452BA;
+    text-decoration: underline #9452BA;
+  }
+`;
+
+const HelperIcon = styled.div`
+  margin-right: 10px;
+  border-right: 1px #c7cbd4 solid;
```

Comment:
```suggestion
  border-right: 1px ${({theme}) => theme.color.grey80} solid;
```
The value of the color `grey80` in the theme (https://github.com/akeneo/pim-community-dev/blob/master/src/Akeneo/Platform/Bundle/UIBundle/Resources/workspaces/shared/src/theme/theme.ts#L64) does not correspond Design System documentation : https://www.notion.so/akeneo/Colors-0a041a4f33814e578890b9b46857ecb1 . You might replace this value by `grey80: '#c7cbd4',`